apps/vue/src/challenges/stopwatch/index.vue

```diff
@@ -0,0 +1,120 @@
+<template>
+    <div class="stopwatch">
+        <div class="timer">
+            <span>{{ time.hour.toString().padStart(2, "0") }}</span>&nbsp;:&nbsp;
+            <span>{{ time.min.toString().padStart(2, "0") }}</span>
+            &nbsp;:&nbsp;
+            <span>{{ time.sec.toString().padStart(2, "0") }}</span>
+        </div>
+        <div class="flex-center">
+            <button @click="startTimer" :disabled="Boolean(intervalID)">Start</button>
+            <button @click="stop" :disabled="!intervalID">Stop</button>
+            <button @click="resetTimer" :disabled="isResetDisabled()">Reset</button>
+        </div>
+    </div>
+</template>
+  
+<script setup lang="ts">
+import { ref } from 'vue';
+
+const time = ref({
+    hour: 0,
+    min: 0,
+    sec: 0,
+});
+
+
+const intervalID = ref(0);
+
+// Local state value update
+const updateTimer = () => {
+    time.value = { ...time.value };
+    let { hour = 0, min = 0, sec = 0 } = { ...time.value };
+
+
+    if (sec < 59) {
+        sec = sec + 1;
+    } else {
+        min = min + 1;
+        sec = 0;
+    }
+
+    if (min === 60) {
+        min = 0;
+        hour = hour + 1;
+    }
+
+    time.value = { min, hour, sec };
+};
+
+// Start button click
+const startTimer = () => {
+    if (intervalID.value) {
+        return;
+    }
+
+    intervalID.value = setInterval(updateTimer, 1000);
+};
+
+// Stop button click
+const stop = () => {
+    if (!intervalID.value) {
+        return;
+    }
+    clearInterval(intervalID.value);
+    intervalID.value = 0;
+};
+
+// Reset button click
+const resetTimer = () => {
+    clearInterval(intervalID.value);
+    intervalID.value = 0;
+
+    time.value = {
+        hour: 0,
+        min: 0,
+        sec: 0,
+    };
+};
+
+// determine disabled state of reset button
+const isResetDisabled = () => {
+    if (!intervalID) {
+        return true
+    }
+
+
+    const { hour = 0, min = 0, sec = 0 } = time.value;
+    return !(hour > 0 || min > 0 || sec > 0);
+
+}
+
+</script>
+  
+<style scoped css="scss">
```

Comment:
I think the `css="scss"` property is not required. See if it works without this property.